app.py

```diff
@@ -68,9 +68,13 @@ def get(self, report_name, query_extension='sql', format_='default'):
         except KeyError:
             result_cls = SQLResultSet
 
-        loader = result_cls(report_name, **request.args.to_dict())
+        loader = result_cls(query_name, **request.args.to_dict())
         query_result = loader.result_set()
 
+        ne_format = request.args.get("ne_format")
```

Comment:
as far as ordering goes, I feel like it make more sense to get the args as a dictionary first, then remove the key that is equivalent to `ne_format` and then pass that dictionary without ne_format to result_cls.
(if that makes sense)